x-pack/plugins/xpack_legacy/server/routes/settings.ts

```diff
@@ -58,9 +58,9 @@ export function registerSettingsRoute({
       const settings =
         (await settingsCollector.fetch(collectorFetchContext)) ??
         settingsCollector.getEmailValueStructure(null);
-      const { cluster_uuid: uuid } = await callAsCurrentUser('info', {
-        filterPath: 'cluster_uuid',
-      });
+
+      const { body } = await collectorFetchContext.esClient.info();
```

Comment:
I think we should keep the `filterPath` option. In the new client it's snake_cased:
```suggestion
      const { body } = await collectorFetchContext.esClient.info({ filter_path: 'cluster_uuid' });
```